ccdproc/image_collection.py

```diff
@@ -58,14 +58,19 @@ class ImageFileCollection(object):
         The filenames are assumed to be in ``location``.
         Default is ``None``.
 
+     hdu: str or int, optional
+         Name/Index of the extension, which specifies the extension to extract data
+         from all files in a given directory.
+         Default is 0.
+
     Raises
     ------
     ValueError
         Raised if keywords are set to a combination of '*' and any other
         value.
     """
     def __init__(self, location=None, keywords=None, info_file=None,
-                 filenames=None):
+                 filenames=None, hdu=0):
```

Comment:
Oh that one is one me. I recommended to use `hdu` because we use `hdu` in [`CCDData.read`](http://ccdproc.readthedocs.io/en/latest/api/ccdproc.fits_ccddata_reader.html#ccdproc.fits_ccddata_reader). I'm not opposed to `ext` but should we change it (not in this PR!) in the reader as well?